### DISCUSSION TITLE
Automation als Kurzbefehl

### DISCUSSION THREAD

**Discussion by gerd2005** [thkl/hap-homematic#408] [Q&A]
Guten Tag,

ich habe folgendes vor. Unsere Außenleuchten sollen um 23:00 Uhr ausgeschaltet werden. Aber nur, wenn die Szene „Gute Nacht“ bereits ausgelöst wurde. Wenn diese nicht ausgelöst wurde, soll das Licht so lange angeschaltet bleiben, bis diese ausgelöst wird. Dies soll verhindern, dass wir Abends auf der Terrasse auf einmal im Dunkeln sitzen.

Ich hatte die Hoffnung, dass ich das mit einer „Wenn-Bedingung“ hinbekomme, in dem ich eine Automation als Kurzbefehl konvertiere. Aber irgendwie steige ich da nicht durch.

Hat jemand für mich vielleicht ein paar Denkanstöße?

Danke und lieben Gruß.

**Comment by blackbasket** [comment]
Hallo Gerd,

zu Automationen in Home habe ich keine Erfahrung. Da könnte ggf Yannik (#373) helfen.

Als Denkanstoß kann ich folgendes anbieten: Falls du über Home in der Automation "Gute Nacht" eine Variable setzt, könntest du einfach ein CCU-Programm bedingt ausführen.

LG,
Marcel
